multi-commander.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "8.3.0.2469",
+    "description": "A multi-tabbed file manager and is an alternative to the standard Windows Explorer",
+    "homepage": "http://multicommander.com/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://multicommander.com/docs/License"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "http://multicommander.com/files/updates/MultiCommander_x64_Portable_(8.3.0.2469).zip",
+            "hash": "sha1:c4c01221b172ae2feef5cb1590536523a9ebe558"
+        },
+        "32bit": {
+            "url": "http://multicommander.com/files/updates/MultiCommander_win32_Portable_(8.3.0.2469).zip",
+            "hash": "sha1:02142ad0c3fe1ab64f7f8b1b6f5b1ad83607e820"
+        }
+    },
+    "shortcuts": [
+        [
+            "MultiCommander.exe",
+            "Multi Commander"
+        ]
+    ],
+    "persist": [
+        "Config",
+		"SessionConfig",
+        "Extensions",
+        "UserData"
+    ],
+    "checkver": {
+        "regex": "version\\s([\\w\\d.]+)\\s\\(build\\s(\\d+)\\)",
+        "replace": "${1}.0.${2}"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "http://multicommander.com/files/updates/MultiCommander_x64_Portable_($version).zip",
+                "hash": {
```

Comment:
Since hash is same for both architectures, it could be generic. Instead of adding it to both architectures, make it `autoupdate.hash` (see: https://github.com/lukesampson/scoop-extras/blob/5b076d3cc50979f9a8e269b4921303622713775f/wavebox.json#L29-L41)

Reply:
Regex do not work in hash. That property is named `find` in hash.

Reply:
In Scoop [Wiki](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#add-hash-to-autoupdate), hash.regex|hash.find are equivalent, and regex works fine for me.

Reply:
@Ash258 fixed that some time ago 😁 
https://github.com/lukesampson/scoop/blob/6060d7cb6d83ab0b1d78c8a02e3d2f7a4e45a647/lib/autoupdate.ps1#L171-L177

https://github.com/lukesampson/scoop/commit/3453487ed65378cc9ba2efc658ed6bc1431ef463

Reply:
@r15ch13 My bad. I did not noticed that commit. Searched it immediatly after message. But for some reason someone who open discussion could not mark it as resolved 🤷‍♀️🤦‍♂️